static/app/views/explore/components/traceItemAttributes/utils.tsx

```diff
@@ -0,0 +1,16 @@
+import {prettifyTagKey} from 'sentry/utils/discover/fields';
+
+function removePrefixes(key: string) {
+  return key.replace('log.', '').replace('sentry.', '');
```

Comment:
Should we update this to explicitly check for the start of the string, instead of any substring? Maybe something like this `key.replace(/^log\.|^sentry\./, '');`

Reply:
Yes, that's a good idea, I'll push the change before merging

Reply:
Done